ts/src/renderer/phaser/classes/PhaserRegion.ts

```diff
@@ -4,13 +4,17 @@ class PhaserRegion extends PhaserEntity {
 	protected entity: Region;
 
 	constructor (
-		scene: Phaser.Scene,
+		scene: GameScene,
 		entity: Region
 	) {
 		super(entity);
 
 		this.gameObject = scene.add.graphics();
 
+		// we don't get depth/layer info from taro,
+		// so it can go in 'debris' layer for now
+		scene.entityLayers[4].add(this.gameObject);
```

Comment:
4 is index for trees and 2 is or debris. seems like either index or comment should be updated

Reply:
fixed